app/utils/pubsub.js

```diff
@@ -49,8 +50,16 @@ var init = ()=>{
 };
 
 var getNewClient = ()=>{
-  var conf = JSON.parse(process.env.REDIS_CONN_JSON || '{}');
-  return new Redis(conf);
+  var options = JSON.parse(process.env.REDIS_CONN_JSON || '{}');
+  if (process.env.REDIS_CERTIFICATE_PATH) {
```

Comment:
consider just checking for the existence of the cert in the mounted config, rather than creating an env var to define where to read from

Reply:
fixed.